cmd/arcstat/arcstat.in

```diff
@@ -441,73 +441,73 @@ def calculate():
 
     v = dict()
     v["time"] = time.strftime("%H:%M:%S", time.localtime())
-    v["hits"] = d["hits"] / sint
-    v["miss"] = d["misses"] / sint
+    v["hits"] = d["hits"] // sint
```

Comment:
To make behavior consistent between python2 and python3: `from __future__ import division` at the top would be helpful

Reply:
That would make it consistent in the opposite direction - python2 would output the wrong values as well.

That's unless we continue replacing `/` with `//`, but then we wouldn't be using `__future__.division` at all.

Reply:
Explicitly requesting floor division with `//` seems like a very reasonable way to handle this.  For reference, here's the PEP-238 change describing the API change.

https://www.python.org/dev/peps/pep-0238/#api-changes